pkg-config output seems wrong

`-losmgpsmap-1.` is not a typo by me but the output of pkg-config. As expected, my project does not compile unless I manually write -losmgpsmap-1.0. Error message is: `/usr/bin/ld: cannot find -losmgpsmap-1.: No such file or directory`. Installed by `sudo apt install libosmgpsmap-1.0-dev`. If it helps, I'm using Xubuntu 22.04.

```
pkg-config --libs osmgpsmap-1.0
-L/usr/local/lib -losmgpsmap-1. -lgtk-3 -lgdk-3 -lpangocairo-1.0 -lpango-1.0 -lharfbuzz -latk-1.0 -lcairo-gobject -lcairo -lgdk_pixbuf-2.0 -lsoup-2.4 -Wl,--export-dynamic -lgmodule-2.0 -pthread -lgio-2.0 -lgobject-2.0 -lglib-2.0
```
I would happily help fixing this, but I'm not sure what exactly is causing the error.

--- a/osmgpsmap-1.0.pc.in
+++ b/osmgpsmap-1.0.pc.in
@@ -7,5 +7,5 @@ Name: @PACKAGE_NAME@
 Description: Moving map widget using openstreet map data
 Version: @PACKAGE_VERSION@
 Requires: gtk+-3.0 libsoup-3.0
-Libs: -L${libdir} -losmgpsmap-1.
-Cflags: -I${includedir}/osmgpsmap-1.2
+Libs: -L${libdir} -losmgpsmap-1.0
+Cflags: -I${includedir}/osmgpsmap-1.0

